cellprofiler/modules/trackobjects.py

```diff
@@ -712,8 +734,50 @@ def create_settings(self):
             </ul></dd>
             </dl>''' % globals())
 
+        self.average_cell_diameter = cps.Float(
+            "Average cell diameter in pixels",
+            35.0, minval=5, doc='''\
+            %(ONLY_IF_FOLLOWNEIGHBORS)s<br>
+            The average cell diameter is used to scale many %(TM_FOLLOWNEIGHBORS)s algorithm parameters. 
+            Please use e.g. ImageJ to measure the average cell diameter in pixels.
+            ''' % globals()
+        )
+
+        self.advanced_parameters = cps.Binary(
+            'Use advanced configuration parameters', False, doc="""
+            %(ONLY_IF_FOLLOWNEIGHBORS)s<br>
+            Do you want to use advanced parameters to configure plugin? They allow for more flexibility,
```

Comment:
> They allow for more flexibility, however usually the default parameters suffice.

Suggestion: "The default values should be sufficient in most cases. You may want to use advanced parameters when X or Y".

Where X or Y could be something like, "cells are incorrectly marked missing between frames", etc. It would be helpful to provide an example or two where the user may benefit from using the advanced parameters.